src/auth/internal/utils.ts

```diff
@@ -24,34 +12,27 @@ export function inferEmailLoginFromURL(featheryClient: Client) {
   else return emailLoginFirebase(featheryClient);
 }
 
-export function inferAuthLogout() {
-  let logout;
+export async function inferAuthLogout() {
+  // logout may not have a value in certain cases, i.e. stytch is already logged out so there is no jwt
+  if (!authState.client) return;
+
   if (isAuthStytch()) {
-    logout = () => getAuthClient().session.revoke();
+    await authState.client.session.revoke();
   } else if (global.firebase) {
-    logout = () => global.firebase.auth().signOut();
+    await authState.client.auth().signOut();
   }
 
-  // logout may not have a value in certain cases, i.e. stytch is already logged out so there is no jwt
-  if (!logout) return;
-
-  logout().then(() => {
-    authState.onLogout();
-    authState.authPhoneNumber = '';
-    authState.authEmail = '';
-    authState.setAuthId('');
-  });
+  authState.onLogout();
+  authState.authPhoneNumber = '';
+  authState.authEmail = '';
+  authState.setAuthId('');
 }
 
 export function isAuthStytch() {
-  const authClient = getAuthClient();
-  if (!authClient) return;
-  const isAuthClientStytch = Object.getOwnPropertySymbols(authClient)
+  if (!authState.client) return;
+  return Object.getOwnPropertySymbols(authState.client)
```

Comment:
I'll have to double check if this works. we are still using the old version of stytch, so can't remember if the old one has these symbols